sormas-backend/src/test/java/de/symeda/sormas/backend/caze/CaseFacadeEjbTest.java

```diff
@@ -1608,102 +1608,7 @@ public void testGetAllActiveCasesIncludeExtendedChangeDateFiltersSample() throws
 		sample.setComment("one comment");
 		getSampleFacade().saveSample(sample);
 
-		assertEquals(0, getCaseFacade().getAllActiveCasesAfter(date).size());
-		assertEquals(1, getCaseFacade().getAllActiveCasesAfter(date, true).size());
-	}
-
-	@Test
```

Comment:
Why have these tests become obsolete?

Reply:
CaseService.getAllActiveCasesAfter with `boolean includeExtendedChangeDateFilters` that is never used outside of tests (introduced with #2059, partly removed by #2674).